utilities/automake.mk

```diff
@@ -1,9 +1,12 @@
 bin_PROGRAMS += \
 	utilities/ovs-appctl \
-	utilities/ovs-testcontroller \
 	utilities/ovs-dpctl \
 	utilities/ovs-ofctl \
 	utilities/ovs-vsctl
+if !P4OVS
+bin_PROGRAMS += utilities/ovs-testcontroller
```

Comment:
Perhaps add a brief comment saying why we don't build `ovs-testcontroller` in P4OVS mode? The reason's not obvious, and there may be questions if we try to upstream the change without an explanation.

Reply:
I had added an explanation in the commit message but added a comment and TODO here. I think we should fix this before upstreaming as most likely we will get pushback if we upstream this